### DISCUSSION TITLE
My opinion on CSD

### DISCUSSION THREAD

**Discussion by Secret-chest** [linuxmint/discussions#21] [Ideas]
I would like CSD to be used only when there are a few (max. 10) commands, not when menubars were formerly used. We can't turn 7 menus into one.

**Comment by axel358** [comment]
I agree. I personally like CSD and the potential they bring. But i've come to realize that some apps are to complex to use one. I prefer having menu bars than removing features just to make the menu smaller. However i personally believe there are some Mint apps that could benefit from CSD, like mint report, here is [my proposal](https://github.com/linuxmint/mintreport/pull/91)

**Comment by alameth** [comment]
I'm disabled. When I install a new desktop OS, I need to spend several days tweaking to get the system where I can use it fluidly and without pain. It involves a fair amount of CSS, hand editing config files, etc.; but at least it's possible. It's a big reason I'm a Linux user: the desktop is far more configurable than any other O.S.

CSD makes the system much harder me, and I believe for others in my situation. At the moment I'm struggling with pix, which is perhaps the single most important application in my entire workflow, and really pushing the limits of what I can handle. (I'd love to know what effort would be required to restore the old menus to pix.)

**Comment by Jeremi360** [comment]
I hate idea of so called hambuger menus that more and more apps try to use now.
For me we should just use CSD to merge title bar with menu bar and toolbar if possible.
Here is my quick mockup how nemo could look like (menu icon in up left corner is just classic window menu):
![nemo](https://github.com/linuxmint/discussions/assets/1294815/899bf5d0-50dd-4ffa-86a0-a22ce1e78b4e)
I propably would already fork some apps to this way if they wasn't written in C/C++ - I hate those langues, we as humanity should abandon those 2 years ago.
